.github/ISSUE_TEMPLATE.md

```diff
@@ -13,20 +13,25 @@ is the best place for getting support.
 
 [Description of the issue]
 
-### Steps to Reproduce (if issue is bug)
+#### Expected behavior (if applicable):
```

Comment:
I like this change because it implies that Improvements should have an expected behavior (and they should!)